iam/keycloak/examples/docker-compose.yml

```diff
@@ -0,0 +1,141 @@
+      DB_ADDR: postgres
+      DB_PORT: 5432
+      DB_DATABASE: keycloak
+      DB_USER: postgres
+      DB_PASSWORD: postgres
+      KEYCLOAK_LOGLEVEL: INFO
+      ROOT_LOGLEVEL: INFO
+      # PROXY_ADDRESS_FORWARDING: "true" # When running Keycloak behind a proxy, you will need to enable proxy address forwarding
+      # KEYCLOAK_STATISTICS: all
+      # DEBUG: "true"
+      # DEBUG_PORT: "*:8787"
+    ports:
+      - "8080:8080"
+      - "8443:8443"
+      # - "8787:8787"
+
+  postgres:
+    image: dojot/postgres:9.5.21-alpine
+    restart: always
+    environment:
+      POSTGRES_DB: keycloak
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "postgres"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    volumes:
+      - postgres-volume:/var/lib/postgresql/data
+    logging:
+      driver: json-file
+      options:
+        max-size: 20m
+        max-file: '5'
+
+  pgadmin:
+    image: dpage/pgadmin4
+    environment:
+      PGADMIN_DEFAULT_EMAIL: admin@pgadmin.com
+      PGADMIN_DEFAULT_PASSWORD: admin
+    depends_on:
+      - postgres
+    restart: always
+    ports:
+      - 5050:80
+    volumes:
+      - pgadmin-volume:/var/lib/pgadmin
+
+  kafka:
+    image: confluentinc/cp-kafka:5.5.0
+    depends_on:
+      - zookeeper
+    restart: always
+    hostname: "kafka"
+    environment:
+      KAFKA_BROKER_ID: "1"
+      KAFKA_ZOOKEEPER_CONNECT: "zookeeper:2181"
+      KAFKA_ADVERTISED_LISTENERS: "PLAINTEXT://:9092"
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: "1"
+      KAFKA_TRANSACTION_STATE_LOG_MIN_ISR: "1"
+      KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR: "1"
+      KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS: "0"
+      KAFKA_LOG_RETENTION_MINUTES: "30"
+      KAFKA_LOG_SEGMENT_BYTES: "262144000"
+      KAFKA_LOG_RETENTION_CHECK_INTERVAL_MS: "30000"
+    volumes:
+      - kafka-volume:/var/lib/kafka/data
+      - kafka-secrets-volume:/etc/kafka/secrets
+    logging:
+      driver: json-file
+      options:
+        max-size: 20m
+        max-file: '5'
+
+  zookeeper:
+    image: "confluentinc/cp-zookeeper:5.5.0"
+    restart: always
+    environment:
+      ZOOKEEPER_REPLICAS: "1"
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_ELECTION_PORT: 3888
+      ZOOKEEPER_SERVER_PORT: 2888
+      ZOOKEEPER_HEAP_SIZE: "2G"
+      ZOOKEEPER_INIT_LIMIT: "5"
+      ZOOKEEPER_LOG_LEVEL: "INFO"
+      ZOOKEEPER_MAX_CLIENT_CNXNS: "100"
+      ZOOKEEPER_MAX_SESSION_TIMEOUT: "40000"
+      ZOOKEEPER_MIN_SESSION_TIMEOUT: "4000"
+      ZOOKEEPER_PURGE_INTERVAL: "0"
+      ZOOKEEPER_SNAP_RETAIN_COUNT: "3"
+      ZOOKEEPER_SYNC_LIMIT: "10"
+      ZOOKEEPER_TICK_TIME: "2000"
+    volumes:
+      - zookeeper-volume:/var/lib/zookeeper/data
+      - zookeeper-log-volume:/var/lib/zookeeper/log
+      - zookeeper-secrets-volume:/etc/zookeeper/secrets
+    logging:
+      driver: json-file
+      options:
+        max-size: 20m
+        max-file: '5'
+
+  kafdrop:
```

Comment:
Maybe you can create a readme for this example and explain how to check the kafka message in kafdrop and what needs to be done in the keycloak to create realms.

Reply:
done!